README.rst

```diff
@@ -146,6 +147,7 @@ Command Line Interface
       --no-fill-borders     Do not add borders to fill image.
       --zoom-offset N       Offset zoom level by N to fit unprojected images to
                             square maps. Defaults to 0.
+      --png8                specifies number of colors in palette. value range from 2 to 256.
```

Comment:
@vijaymadhavan Could you specify what the default behaviour for this argument is here?

Reply:
sure. will do